utils/redis_storage.ts

```diff
@@ -1,6 +1,7 @@
-import { Either, isLeft, left, right, toError } from "fp-ts/lib/Either";
-import { fromNullable, Option } from "fp-ts/lib/Option";
-import { fromEither, TaskEither, tryCatch } from "fp-ts/lib/TaskEither";
+import * as E from "fp-ts/lib/Either";
+import { pipe } from "fp-ts/lib/function";
+import * as O from "fp-ts/lib/Option";
+import * as TE from "fp-ts/lib/TaskEither";
 import { RedisClient } from "redis";
```

Comment:
Most of the fuctions defined in `redis_storage.ts` are equivalent to the ones defined in `io-function-cgn`, and they seems o be quite general-purpose. 
Maybe we can think about moving them to a common library?

Reply:
Absolutely yes, we have to refactor this when we move shared code into a common library 👍